ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -9,6 +9,34 @@ <h1>Speaking Practice View</h1>
 <p id="text-for-timer"><span id="text-timer"> </span> </p>
 <progress id="timer-bar"></progress>
 
+<!-- original -->
```

Comment:
This should not be referencing original vs reformulation, since we will only be showing originals